README.md

```diff
@@ -279,13 +279,26 @@ clever drain remove [--alias <alias>] <DRAIN-ID>
 
 Where `DRAIN-TYPE` is one of:
 
-- `TCPSyslog`: for TCP syslog endpoint ;
-- `UDPSyslog`: for UDP syslog endpoint ;
+- `TCPSyslog`: for TCP syslog endpoint;
+- `UDPSyslog`: for UDP syslog endpoint;
 - `HTTP`: for TCP syslog endpoint (note that this endpoint has optional username/passwordparameters as HTTP Basic Authentication);
-- `ElasticSearch`: for ElasticSearch endpoint (note that this endpoint requires username/password parameters as HTTP Basic Authentication).
+- `ElasticSearch`: for ElasticSearch endpoint (note that this endpoint requires username/password parameters as HTTP Basic Authentication);
+- `DatadogHTTP`: for Datadog endpoint (note that this endpoint needs your Datadog API Key).
+
+#### ElasticSearch logs drains
 
 ElasticSearch drains use the Elastic bulk API. To match this endpoint, specify `/_bulk` at the end of your ElasticSearch endpoint.
 
+#### Datadog logs drains
+
+Datadog has two zones, EU and COM. An account on one zone is not available on the other, make sure to target the good EU or COM intake endpoint.
+
+To create a [Datadog](https://docs.datadoghq.com/api/?lang=python#send-logs-over-http) drain, you just need to use:
+
+```sh
+clever drain create DatadogHTTP "https://http-intake.logs.datadoghq.com/v1/input/<API_KEY>?ddsource=clevercloud&service=<SERVICE>&host=<HOST>"
```

Comment:
host is not mandatory, since you forward `@source_host` property.

In the datadog pipeline configuration, you can then map `@source_host` as host property

Reply:
Will add this comment.